Expensify/App
# Fix - Reduce the haptic feedback vibration to a short bip

cc: @Jag96 Creating a draft PR. Looking forward for your feedback!
### Details

Use the lib [`Expensify/react-native-haptic-feedback`](https://github.com/Expensify/react-native-haptic-feedback) for haptic feedback.
Currently it points to the latest commit on my [PR](https://github.com/Expensify/react-native-haptic-feedback/pull/1) for Expensify/react-native-haptic-feedback.

#### Android
Create a new file: components/PressableWithSecondaryInteraction/index.android.js
Call `ReactNativeHapticFeedback.trigger('effectHeavyClick');` for a long press.
#### iOS
Create a new file: components/PressableWithSecondaryInteraction/index.ios.js
Call `ReactNativeHapticFeedback.trigger('selection');` for a long press.
### Fixed Issues

$ https://github.com/Expensify/App/issues/4355

### Tests / QA

1. Navigate to a conversation on mobile (iOS, Android).
2. Long press on a message.
3. Verify that the haptic feeback is same as slack's long press on a message.

### Tested On

### Screenshots
Not Applicable
